libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1035,6 +1035,25 @@ size_t WaveTrack::NIntervals() const
    return mClips.size();
 }
 
+auto WaveTrack::GetWideClip(size_t iInterval) -> IntervalHolder
```

Comment:
Looks like this method is intended for calls on leaders ; if so, an assert would clarify that.